### ISSUE TITLE
Too much padding around code blocks

### ISSUE THREAD

**Issue by rak-phillip** [rancher-sandbox/rancher-desktop#3268]
- [ ] The code block box is too tall. The boxes on successive lines are almost touching; they need visible separation.
- [ ] The code block box is also a tiny bit too wide IMO[1](https://github.com/rancher-sandbox/rancher-desktop/pull/3251#user-content-fn-1-fbe9102fe864b4ddd90304397a6f97b0).

**Comment by kwwii** [comment]
I don't think that changing the line height or such would easily fix the appearance of them overlapping. In general, I think that the current usage of codeblocks could be improved from a UX standpoint.

For instance, instead of the current description of the symlink I would rephrase it so that the codeblock is one distinct piece of text seperate from the rest:

Symlink missing. Run the following command in a terminal:
<codeblock>ln -s ~/docker/cli-plugins/docker-buildx ~/.rd/bin/docker-buildx</codeblock>

Environment Variable missing. Run the following command in a terminal:
<codeblock>export PATH=$PATH:~/.rd/bin</codeblock>

This way, the user clearly sees what is happening and has a simple "one-liner" to copy (and possibly learn from)
